api/test/integration/mapping/integration_test_api_endpoints.json

```diff
@@ -2829,11 +2354,7 @@
                     "test_agent_POST_endpoints.tavern.yaml",
                     "test_agent_PUT_endpoints.tavern.yaml",
                     "test_cluster_endpoints.tavern.yaml",
-                    "test_experimental_endpoints.tavern.yaml",
-                    "test_security_DELETE_endpoints.tavern.yaml",
-                    "test_security_GET_endpoints.tavern.yaml",
-                    "test_security_POST_endpoints.tavern.yaml",
-                    "test_security_PUT_endpoints.tavern.yaml"
+                    "test_experimental_endpoints.tavern.yaml"
```

Comment:
These changes are not correct. The `integration_test_api_endpoints.json` should not be modified manually. If you want to keep experimental tests in the basic suite, they should remain in the `_test_mapping.py` script:
![image](https://github.com/wazuh/wazuh/assets/23361101/6df4a349-61cb-4d58-b99c-a4aea80fc8e5)

Then, a new .json file should be created using the script. If you modify the json manually and not the script, next time someone needs to create a new json, these manual changes will disappear.